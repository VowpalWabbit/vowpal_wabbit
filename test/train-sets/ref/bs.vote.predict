0.000000 0.000000 0.000000
0.000000 0.000000 0.165033
0.000000 0.000000 0.148058
0.000000 0.000000 0.056914
0.000000 0.000000 0.055368
0.000000 0.000000 0.108588
0.000000 0.063254 0.097981
0.000000 0.195273 0.207024
0.000000 0.080428 0.130979
0.000000 0.161919 0.224062
0.000000 0.159888 0.292871
0.000000 0.130859 0.262355
0.000000 0.055605 0.234676
0.000000 0.154614 0.291912
0.000000 0.100355 0.186234
0.000000 0.271194 0.359921
0.000000 0.195800 0.297624
0.000000 0.306360 0.368946
0.000000 0.394553 0.488798
0.000000 0.129983 0.259065
0.000000 0.072150 0.138995
0.000000 0.287308 0.556645
0.000000 0.208845 0.383221
0.000000 0.219426 0.265090
0.000000 0.112642 0.136059
0.000000 0.296746 0.493424
0.000000 0.139164 0.244939
0.000000 0.212882 0.358598
0.000000 0.249371 0.381788
0.000000 0.235390 0.377699
0.000000 0.354900 0.538839
0.000000 0.118062 0.259427
0.000000 0.137846 0.214422
0.000000 0.270229 0.359664
0.000000 0.332286 0.544358
0.000000 0.238639 0.520331
0.000000 0.226335 0.376857
0.000000 0.286227 0.427360
0.000000 0.363994 0.460808
1.000000 0.439406 0.570028
0.000000 0.213245 0.394100
0.000000 0.265211 0.433544
0.000000 0.334582 0.457471
0.000000 0.210034 0.333600
0.000000 0.265901 0.353179
0.000000 0.121864 0.280419
0.000000 0.338720 0.662083
0.000000 0.385756 0.540913
0.000000 0.066106 0.284537
0.000000 0.140253 0.192532
0.000000 0.224389 0.297352
0.000000 0.274979 0.322848
0.000000 0.338528 0.369949
0.000000 0.293847 0.427852
0.000000 0.434942 0.525146
1.000000 0.535901 0.705668
0.000000 0.391931 0.458663
0.000000 0.255923 0.343710
0.000000 0.380002 0.476756
0.000000 0.279792 0.381307
0.000000 0.266566 0.319867
0.000000 0.355619 0.443840
1.000000 0.399055 0.614496
0.000000 0.178605 0.333045
0.000000 0.434491 0.492606
0.000000 0.225289 0.316254
0.000000 0.301119 0.430593
1.000000 0.610695 0.800837
0.000000 0.307809 0.441826
0.000000 0.349509 0.558426
0.000000 0.386816 0.417430
0.000000 0.401468 0.566613
0.000000 0.180458 0.388247
0.000000 0.205228 0.385539
0.000000 0.206109 0.522599
0.000000 0.258802 0.414362
0.000000 0.170788 0.471904
1.000000 0.328829 0.588173
0.000000 0.125418 0.154364
0.000000 0.141366 0.238340
0.000000 0.211104 0.259811
0.000000 0.219673 0.348892
1.000000 0.538936 0.596301
0.000000 0.244495 0.371420
0.000000 0.166359 0.436095
0.000000 0.233814 0.306213
0.000000 0.214012 0.236293
1.000000 0.367507 0.653899
1.000000 0.536052 0.687675
0.000000 0.200344 0.393470
0.000000 0.074324 0.139967
0.000000 0.253176 0.503452
1.000000 0.712092 0.911009
0.000000 0.148967 0.233552
1.000000 0.420521 0.525129
0.000000 0.281586 0.449643
0.000000 0.415734 0.527373
0.000000 0.180867 0.257021
0.000000 0.287143 0.435619
0.000000 0.245233 0.386528
1.000000 0.542068 0.817527
1.000000 0.407375 0.554181
0.000000 0.307080 0.433866
0.000000 0.412964 0.456685
1.000000 0.393871 0.768310
0.000000 0.466066 0.559906
0.000000 0.124144 0.193973
0.000000 0.322430 0.424669
0.000000 0.151825 0.278742
1.000000 0.301222 0.603838
1.000000 0.522079 0.693242
1.000000 0.298500 0.648683
0.000000 0.383050 0.521281
0.000000 0.219588 0.304680
0.000000 0.366373 0.501866
0.000000 0.336869 0.591129
0.000000 0.148992 0.268347
1.000000 0.414344 0.601728
0.000000 0.152421 0.557340
0.000000 0.346790 0.408564
0.000000 0.283772 0.445531
0.000000 0.422995 0.541149
0.000000 0.383448 0.464324
0.000000 0.331814 0.489881
1.000000 0.482536 0.670951
0.000000 0.310524 0.511148
0.000000 0.306036 0.407844
1.000000 0.472411 0.723238
0.000000 0.086951 0.354961
0.000000 0.287749 0.554533
0.000000 0.261021 0.396569
1.000000 0.456366 0.835763
0.000000 0.161112 0.237505
0.000000 0.235775 0.305934
0.000000 0.300791 0.418951
0.000000 0.189224 0.323355
1.000000 0.510222 0.586318
0.000000 0.336993 0.492896
1.000000 0.634971 0.807615
0.000000 0.289937 0.400814
0.000000 0.408571 0.462840
0.000000 0.207687 0.290693
1.000000 0.702547 0.775726
1.000000 0.763069 0.883828
0.000000 0.430606 0.536159
1.000000 0.458201 0.627050
1.000000 0.421077 0.602258
1.000000 0.783578 0.893522
0.000000 0.265009 0.486779
0.000000 0.313970 0.442856
1.000000 0.341150 0.744139
0.000000 0.266393 0.388470
1.000000 0.588819 0.719345
0.000000 0.478017 0.570976
1.000000 0.535712 0.753140
1.000000 0.540066 0.643340
1.000000 0.788869 0.903525
1.000000 0.500941 0.624359
1.000000 0.672801 0.870738
1.000000 0.440563 0.608410
1.000000 0.408206 0.689143
0.000000 0.226362 0.414947
0.000000 0.244450 0.602926
1.000000 0.651315 0.706431
0.000000 0.294774 0.356835
0.000000 0.393913 0.481877
1.000000 0.728997 0.908197
0.000000 0.326370 0.616926
1.000000 0.426752 0.529047
1.000000 0.399556 0.781778
1.000000 0.650429 0.724167
0.000000 0.265291 0.379296
0.000000 0.214054 0.309486
0.000000 0.315409 0.409288
0.000000 0.123357 0.179197
1.000000 0.465758 0.637765
0.000000 0.283083 0.327480
1.000000 0.780997 0.982180
1.000000 0.539124 0.687386
1.000000 0.697587 0.776066
0.000000 0.395733 0.522453
0.000000 0.362856 0.415778
1.000000 0.443249 0.594379
0.000000 0.280466 0.441391
0.000000 0.162939 0.431504
0.000000 0.281772 0.562431
1.000000 0.601315 0.829984
1.000000 0.536297 0.689820
0.000000 0.198638 0.376431
1.000000 0.508416 0.790084
1.000000 0.450380 0.715270
0.000000 0.091925 0.486855
0.000000 0.210002 0.321692
0.000000 0.259601 0.316226
0.000000 0.257007 0.385543
0.000000 0.454336 0.708163
0.000000 0.447638 0.577024
0.000000 0.282050 0.458769
0.000000 0.253884 0.458618
1.000000 0.386388 0.674689
