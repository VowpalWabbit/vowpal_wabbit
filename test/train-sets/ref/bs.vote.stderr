final_regressor = models/bs.vote.model
predictions = bs.vote.predict
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/0001.dat
num sources = 1
Enabled reductions: gd, scorer-identity, bootstrap
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       51
0.500000 0.000000            2            2.0   0.0000   0.0000      104
0.250000 0.000000            4            4.0   0.0000   0.0000      135
0.250000 0.250000            8            8.0   0.0000   0.0000      146
0.312500 0.375000           16           16.0   1.0000   0.0000       24
0.343750 0.375000           32           32.0   0.0000   0.0000       32
0.343750 0.343750           64           64.0   0.0000   0.0000       61
0.359375 0.375000          128          128.0   1.0000   1.0000      106

finished run
number of examples = 200
weighted example sum = 200.000000
weighted label sum = 91.000000
average loss = 0.310000
best constant = 0.455000
best constant's loss = 0.247975
total feature number = 15482
